bittensor/extrinsics/root.py

```diff
@@ -177,7 +180,7 @@ def set_root_weights_extrinsic(
             weight_uids, weight_vals = weight_utils.convert_weights_and_uids_for_emit(
                 netuids, weights
             )
-            success, error_message = subtensor._do_set_weights(
+            success, error_message = subtensor._do_set_root_weights(
```

Comment:
PEP 8: Access to a protected member `_do_set_root_weights` of a class `Subtensor()`

Reply:
Since this is back-porting master, i'll make this change in upcoming PRs in staging.